### PR TITLE
RHDEVDOCS-4137 - Added 1.5.1 and 1.5.2 release notes

### PR DESCRIPTION
**Pull request by DebarghoGhosh**
Aligned team: Dev Tools
OCP version for cherry-picking: enterprise-4.10, 4.11

JIRA issue:
https://issues.redhat.com/browse/RHDEVDOCS-4137

Preview pages: No preview generated due to Netlify issues.

SME review: @iam-veeramalla 
Peer-review: @rolfedh 
QE review: @varshab1210

### CODE REVIEW

File: modules/gitops-release-notes-1-5-1.adoc [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
```
Comment (rolfedh) [inline modules/gitops-release-notes-1-5-1.adoc:4]:
```suggestion
:_content-type: REFERENCE
```
Every file must have a content type attribute.

File: modules/gitops-release-notes-1-5-1.adoc [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+= Release notes for {gitops-title} 1.5.1
+
+{gitops-title} 1.5.1 is now available on {product-title} 4.8, 4.9, and 4.10.
+
+[id="fixed-issues-1-5-1_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, an unauthenticated user can craft a JWT token while Argo CD's anonymous access is enabled and get full access to the Argo CD instance. This issue is fixed now. link:https://bugzilla.redhat.com/show_bug.cgi?id=2081686[CVE-2022-29165]
```
Comment (rolfedh) [inline modules/gitops-release-notes-1-5-1.adoc:15]:
```suggestion
* Before this update, an unauthenticated user was able to craft a JWT token while Argo CD's anonymous access is enabled and get full access to the Argo CD instance. This issue is fixed now. link:https://bugzilla.redhat.com/show_bug.cgi?id=2081686[CVE-2022-29165]
```

File: modules/gitops-release-notes-1-5-1.adoc [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+* Before this update, an unauthenticated user can display error messages on the login screen when SSO in enabled. This issue is now fixed. link:https://bugzilla.redhat.com/show_bug.cgi?id=2081689[CVE-2022-24905]
```
Comment (rolfedh) [inline modules/gitops-release-notes-1-5-1.adoc:17]:
```suggestion
* Before this update, an unauthenticated user was able to display error messages on the login screen while SSO was enabled. This issue is now fixed. link:https://bugzilla.redhat.com/show_bug.cgi?id=2081689[CVE-2022-24905] 
```

File: modules/gitops-release-notes-1-5-1.adoc [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, an unauthenticated user can craft a JWT token while Argo CD's anonymous access is enabled and get full access to the Argo CD instance. This issue is fixed now. link:https://bugzilla.redhat.com/show_bug.cgi?id=2081686[CVE-2022-29165]
+
+* Before this update, an unauthenticated user can display error messages on the login screen when SSO in enabled. This issue is now fixed. link:https://bugzilla.redhat.com/show_bug.cgi?id=2081689[CVE-2022-24905] 
+
+* Before this update, all unpatched versions Argo CD starting v7.0 were vulnerable to a symlink following bug. This allowed an unauthorized user with repository write
+access to leak sensitive files from Argo CD's repo-server. This issue is now fixed. link:https://bugzilla.redhat.com/show_bug.cgi?id=2081686[CVE-2022-24904]
```
Comment (rolfedh) [inline modules/gitops-release-notes-1-5-1.adoc:20]:
```suggestion
* Before this update, all unpatched versions Argo CD starting v7.0 were vulnerable to a symlink-following bug. This allowed an unauthorized user with repository write access to leak sensitive files from Argo CD's repo-server. This issue is now fixed. link:https://bugzilla.redhat.com/show_bug.cgi?id=2081686[CVE-2022-24904]
```

File: modules/gitops-release-notes-1-5-2.adoc [hunk 5]
```diff
@@ -0,0 +1,15 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
```
Comment (rolfedh) [inline modules/gitops-release-notes-1-5-2.adoc:4]:
```suggestion
:_content-type: REFERENCE
```

File: modules/gitops-release-notes-1-5-1.adoc (hunk 4 above)
Comment (rolfedh) [reply modules/gitops-release-notes-1-5-1.adoc:20]: Note: `symlink-following` and `...fixed.  link:...`

File: modules/gitops-release-notes-1-5-2.adoc [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+= Release notes for {gitops-title} 1.5.2
+
+{gitops-title} 1.5.2 is now available on {product-title} 4.8, 4.9, and 4.10.
+
+[id="fixed-issues-1-5-2_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, images referred by the `redhat-operator-index` were missing. This issue is now fixed.  link:https://issues.redhat.com/browse/GITOPS-2036[GITOPS-2036]
```
Comment (rolfedh) [inline modules/gitops-release-notes-1-5-2.adoc:15]:
```suggestion
* Before this update, images referenced by the `redhat-operator-index` were missing. This issue is now fixed.  link:https://issues.redhat.com/browse/GITOPS-2036[GITOPS-2036]
```

**Review by rolfedh — CHANGES_REQUESTED** [review]
Please apply the suggested changes.

File: modules/gitops-release-notes-1-5-1.adoc (hunk 2 above)
Comment (rolfedh) [reply modules/gitops-release-notes-1-5-1.adoc]: Please review this suggestion.

File: modules/gitops-release-notes-1-5-1.adoc [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+= Release notes for {gitops-title} 1.5.1
+
+{gitops-title} 1.5.1 is now available on {product-title} 4.8, 4.9, and 4.10.
+
+[id="fixed-issues-1-5-1_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Before this update, an unauthenticated user was able to craft a JWT token while Argo CD's anonymous access is enabled and get full access to the Argo CD instance. This issue is fixed now. link:https://bugzilla.redhat.com/show_bug.cgi?id=2081686[CVE-2022-29165]
```
Comment (rolfedh) [inline modules/gitops-release-notes-1-5-1.adoc:17]: Please see https://github.com/openshift/openshift-docs/pull/47155#discussion_r908350064

File: modules/gitops-release-notes-1-5-1.adoc [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+
+* Before this update, an unauthenticated user was able to display error messages on the login screen while SSO was enabled. This issue is now fixed. link:https://bugzilla.redhat.com/show_bug.cgi?id=2081689[CVE-2022-24905] 
+
+* Before this update, all unpatched versions of Argo CD starting v7.0 were vulnerable to a symlink-following bug. This allowed an unauthorized user with repository write access to leak sensitive files from Argo CD's repo-server. This issue is now fixed. link:https://bugzilla.redhat.com/show_bug.cgi?id=2081686[CVE-2022-24904]
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply modules/gitops-release-notes-1-5-1.adoc:21]:
```suggestion
* Before this update, all unpatched versions of Argo CD v7.0 and later were vulnerable to a symlink-following bug. This bug made it possible for an unauthorized user with repository write access to leak sensitive files from Argo CD's repo-server. This issue is now fixed. link:https://bugzilla.redhat.com/show_bug.cgi?id=2081686[CVE-2022-24904]
```

**Review by rolfedh — APPROVED** [review]

**Comment by DebarghoGhosh** [comment]
I have followed the [OpenShift Peer Review Checklist > (for writers)](https://docs.google.com/document/d/1WN7k72PKxcPA__erp5TK-CefmS_sVwE3r40PvpmNk3E/edit#heading=h.bp4j7064auhh)

**Pull request merged**